db/migrate/20230504181957_create_likes.rb

```diff
@@ -1,7 +1,6 @@
 class CreateLikes < ActiveRecord::Migration[7.0]
   def change
     create_table :likes do |t|
```

Comment:
* As shown in the ERD diagram you should have a foreign key that will create author_id and a corresponding index
![image](https://user-images.githubusercontent.com/52497246/236378887-b885b55c-5c70-451e-8703-7d11eb15bdef.png)

Reply:
As requested by the previous CR, you have a foreign_key in your migrations that links tables to each other. For example in your CreatePost migration, you should have something similar to the following to link the post to a user:
```
`t.references :author, foreign_key: { to_table: 'users' }`
```
- Now using the ERD diagram include a corresponding foreign_key to other migrations.